### PR TITLE
[New Rule] Sensitive Files Compression

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/755

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/linux/credential_access_collection_sensitive_files.toml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+[metadata]
+creation_date = "2020/12/22"
+maturity = "production"
+updated_date = "2020/12/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of a compression utility to collect known files containing sensitive information such as credentials
```
Comment (bm11100) [inline rules/linux/credential_access_collection_sensitive_files.toml:9]:
```suggestion
Identifies the use of a compression utility to collect known files containing sensitive information, such as credentials
```

**Review by bm11100 — APPROVED** [review]
lgtm 🥓

File: rules/linux/credential_access_collection_sensitive_files.toml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "https://www.trendmicro.com/en_ca/research/20/l/teamtnt-now-deploying-ddos-capable-irc-bot-tntbotinger.html",
+]
+risk_score = 47
+rule_id = "6b84d470-9036-4cc0-a27c-6d90bbfe81ab"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Collection", "Credential Access"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
```
Comment (peasead) [inline rules/linux/credential_access_collection_sensitive_files.toml:27]:
```suggestion
event.category:process and event.type:start and
```

File: rules/linux/credential_access_collection_sensitive_files.toml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+ process.name:(zip or tar or gzip or hdiutil) and
```
Comment (peasead) [inline rules/linux/credential_access_collection_sensitive_files.toml:28]:
```suggestion
 process.name:(zip or tar or gzip or hdiutil or 7z) and
```

File: rules/linux/credential_access_collection_sensitive_files.toml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+              /root/.aws/config or 
+              /home/*/.aws/credentials or 
+              /home/*/.aws/config or 
+              /root/.docker/config.json or 
+              /home/*/.docker/config.json or 
+              /etc/group or 
+              /etc/passwd or 
+              /etc/shadow or 
+              /etc/gshadow
+             )
```
Comment (bm11100) [inline rules/linux/credential_access_collection_sensitive_files.toml:57]:
```suggestion
event.category:process and event.type:start and
  process.name:(zip or tar or gzip or hdiutil or 7z) and
  process.args:
    (
      /root/.ssh/id_rsa or
      /root/.ssh/id_rsa.pub or
      /root/.ssh/id_ed25519 or
      /root/.ssh/id_ed25519.pub or
      /root/.ssh/authorized_keys or
      /root/.ssh/authorized_keys2 or
      /root/.ssh/known_hosts or
      /root/.bash_history or
      /etc/hosts or
      /home/*/.ssh/id_rsa or
      /home/*/.ssh/id_rsa.pub or
      /home/*/.ssh/id_ed25519 or
      /home/*/.ssh/id_ed25519.pub or
      /home/*/.ssh/authorized_keys or
      /home/*/.ssh/authorized_keys2 or
      /home/*/.ssh/known_hosts or
      /home/*/.bash_history or
      /root/.aws/credentials or
      /root/.aws/config or
      /home/*/.aws/credentials or
      /home/*/.aws/config or
      /root/.docker/config.json or
      /home/*/.docker/config.json or
      /etc/group or
      /etc/passwd or
      /etc/shadow or
      /etc/gshadow
    )
```

**Review by threat-punter — APPROVED** [review]

**Pull request merged**